angular/_pxb-component-theme.scss

```diff
@@ -112,6 +112,12 @@
         .pxb-drawer-nav-item-active-highlight {
             background-color: map-get($primary, 500);
         }
+        .pxb-drawer-nav-item-rail {
+            &:hover,
+            &:focus {
+                background: mat-color($background, 'hover');
```

Comment:
This is the same way that mat-list-items receive their highlighting, just took this snippet from the @angular/material theme.